public/style/food-trough-panel.css

```
@@ -0,0 +1,265 @@
+
+#food-trough-panel {
+  position: fixed;
+  inset: 0;
+  z-index: 9999;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  opacity: 0;
+  pointer-events: none;
+  transition: opacity 0.3s ease;
+  font-family: 'Georgia', serif;
```

Stylelint (17.12.0)

[error] Expected no quotes around "Georgia" (font-family-name-quotes)

(font-family-name-quotes)